include/swift/Frontend/ModuleInterfaceLoader.h

```diff
@@ -150,6 +150,7 @@ class ModuleInterfaceLoader : public SerializedModuleLoaderBase {
     AccessPathElem ModuleID, StringRef DirPath, StringRef ModuleFilename,
     StringRef ModuleDocFilename,
     StringRef ModuleSourceInfoFilename,
+    SmallVectorImpl<char> *ModuleInterfacePath,
```

Comment:
Yuck, but okay, I see that this is probably the only way to feed it through. Maybe we should make it more generic though: "FoundPath" or "PathForDependencyTracking" or something. What do you think?

Reply:
That sounds reasonable. I could rename it to `FoundModuleInterfacePath`.

Reply:
Oh, I see what you mean. I don't see why making it more generic is useful, because then it becomes less clear what the semantics of that path are. If let's say in the future, we want a bunch of other paths out, maybe we can create a struct? (Actually, I was debating creating a struct to contain all the outparams...)

Reply:
I was thinking it's "overriding file name for debugging, dependency tracking, and other purposes that want to ask for a single path that represents this module", but that's a little long. 😄